rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -91,9 +92,11 @@ bool RSocketStateMachine::resumeServer(
             << "Terminating new connection (" << frameTransport.get() << ")";
     frameTransport->closeWithError(folly::exception_wrapper(
         std::runtime_error("Old connection still exists")));
+    stats_->resumeFailure();
```

Comment:
It looks like the resumption can fail with a few reasons. I am not sure we need to for v1, but it might be beneficial to be able to differentiate the reason of the failure. @somasun ?

Reply:
Right, we can just add resumeFailure() to get things running.  For v2 we can go for more details.